playbook/app/pb_kits/playbook/pb_timestamp/timestamp.rb

```diff
@@ -27,6 +27,15 @@ class Timestamp < Playbook::KitBase
                      values: %w[default elapsed updated],
                      default: "default"
 
+      # Variables to use with pb_time_ago method
+      SECS_PER_MIN  = 60
+      SECS_PER_HOUR = 60 * SECS_PER_MIN
+      SECS_PER_DAY  = 24 * SECS_PER_HOUR
+      SECS_PER_WEEK = 7 * SECS_PER_DAY
+      SECS_PER_MONTH = 4 * SECS_PER_WEEK
+      SECS_PER_YEAR = 12 * SECS_PER_MONTH
+      SECS_PER_CENT = 100 * SECS_PER_YEAR
```

Comment:
I was wondering if we'd ever show "centuries ago", but this is just the limit for time elapsed.